### PR TITLE
Fixed immer vulnerability with force resolutions for transitive dependencies

### PR DESCRIPTION
**Pull request by johanna-adhoc**
### Added force resolution packages to enforce version in transitory dependency

Immer high vulnerability in react-dev-utils(dev dependency) is blocking PR builds. This PR add the force resolution package to force the immer version that does not contain the vulnerability

### Process
- [X] Tests added or updated for change

### Requested feedback
All feedback is welcome.

### CODE REVIEW

**Review by gileswells — APPROVED** [review]
🚀 LGTM

Literally my only question is the difference on number of spaces indenting the `Dockerfile` but that's a non-issue

**Comment by johanna-adhoc** [comment]
> 🚀 LGTM
> 
> Literally my only question is the difference on number of spaces indenting the `Dockerfile` but that's a non-issue

My auto formatter is doing that, it takes the eslint configuration and adds the spaces :/

File: Dockerfile [hunk 1]
```diff
@@ -30,6 +30,11 @@ RUN npm install -g @sentry/cli
 
 COPY package.json package-lock.json ./
 
+# Add write permission to package-lock.json so npm resolutions can set the correct dependency version
+USER root
```
Comment (mjlumetta) [inline Dockerfile:34]: I think this whole file could run as root anyway now if we wanted... but not super important for this PR

**Review by mjlumetta — APPROVED** [review]
LGTM. thank you for fixing this @johanna-adhoc!

**Review by TS000 — APPROVED** [review]
Looks great! Take that immer!

**Pull request merged**